styles.css

```diff
@@ -2,30 +2,499 @@
+
+
+}
+.header-button button{
+    padding: 15px 30px;
+    font-size: 10px;
+    border-radius: 30px;
+    margin: 25px 0;
+    border-style: none;
+}
+
+.header-img{
+position: relative;
+}
+
+.squiggle_line_stripe{
+    position: relative;
+    bottom: -90px;
+    scale: 80%;
+}
+
+
+.arrow-left{
+    border-top: 0 solid transparent;
+    border-bottom: 8em solid transparent;
+    border-right:170em solid var(--primary-background);
+    margin-bottom: -10px;
+}
+
+
+/* --------------- 3.3 Introduction ------------------- */
+
+.introduction {
+    display: flex;
+    background-color: white;
+}
+.bgtext::after {
+    content: "hello";
+    color: var(--secondary-light-background-text);
+    font-weight: bold;
+    font-size: 500px;
+    position: absolute;
+    left: -70px;
+    z-index: 1;
+    top: -100px;
+}
+
+
+.introduction-text {
+    color: var(--secondary-h);
+    justify-content: space-evenly;
+    text-align: center;
+    gap: 35px;
+    font-size: 1.5rem;
+}
+a {
+    text-decoration: none;
+    color: var(--secondary-h);
+}
+.introduction-text p{
+color: var(--primary-text);
+}
+
+
+/* --------------- 3.4 Projects ------------------- */
+.arrow-right {
+    height: 100px;
+    border-bottom: 0 solid transparent;
+    border-top: 8em solid transparent;
+    border-left: 119em solid var(--projects-bonus);
+    margin-top: 10px;
+}
+
+section {
+    background-color:var(--projects-bonus);
+    position: relative;
+}
+
+.flex-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+.flex-container>div {
+    text-align: left;
+    margin: 30px;
+}
+.flex-reverse{
+    flex-direction: row-reverse;
+}
+.rolling-stone {
+flex-grow: 2;
+    width: 200px;
+    height: 200px;
+    z-index: 2;
+}
+
+.rolling-stone>h2 {
+    position: relative;
+}
+.project-bgtext::after {
+    content: "work";
+    color: white;
+    font-weight: bold;
+    font-size: 700px;
+    position: absolute;
+    left: 400px;
+    z-index: 1;
+    top: -120px;
+}
+
+.work-image {
+    width: 100%;
+    min-width: 200px;
+    box-shadow:  0.3em 1.5em 1.4em rgba(0, 0, 0, 0.2);
+}
+
+.image-wrapper {
+    flex-basis: 800px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    z-index: 2;
+}
+
+.squiggle_line {
+    margin: 50px 0;
+}
+.p-view-project {
+    margin: 50px 0;
+    link: pointer;
+    font: small-caps bold 12px/1 sans-serif;
+}
+
+/* --------------- 3.5 Services ------------------- */
+.services-arrow-left {
+    border-top: 0 solid transparent;
+    border-bottom: 8em solid transparent;
+    border-right:170em solid var(--projects-bonus);
+    margin-bottom: -10px;
+}
+
+.services-section {
+    display: flex;
+    background-color: white;
+}
+
+.services-bgtext::after {
+    content: "services";
+    color: var(--secondary-light-background-text);
+    font-weight: bold;
+    font-size: 400px;
+    position: absolute;
+    left: 400px;
+    z-index: 1 ;
```

Comment:
Deze weghalen maakte voor mij niets uit, maar misschien zie ik iets over het hoofd. En had je anders niet beter gebruik kunnen maken van de ::before pseudo in plaats van de ::after? Dan had die z-index misschien niet nodig geweest